README.md

```diff
@@ -187,75 +143,76 @@ At the root of this repository, execute `setup.sh`. It will:
 
 ## Validation
 
-Validating these three deployments is done by simply executing `validate.sh`
-from the root of the directory.
-A successful output will contain:
+Validating these three deployments is done by executing:
+
+```console
+make validate
+```
+
+A successful output will look like this:
```

Comment:
... wil look something like this: (with variables instead of hardcoded ips)

Reply:
There's a comment to that effect underneath the console output. Are you suggested merge them into one comment above?

Reply:
let's keep as is as per our discussion